misc/scripts/build.sh

```diff
@@ -140,6 +139,9 @@ else
     -e "ESX=$2" \
     -e "VM1=$3" \
     -e "VM2=$4" \
+    -e "DS1=$DS1" \
+    -e "VM1NAME=$VM1NAME" \
```

Comment:
I would suggest to use our automation script to deploy testbed on nimbus and test out your changes locally.